server/jetstream_api.go

```diff
@@ -766,8 +767,9 @@ const JSApiConsumerNamesResponseType = "io.nats.jetstream.api.v1.consumer_names_
 type JSApiConsumerListResponse struct {
 	ApiResponse
 	ApiPaged
-	Consumers []*ConsumerInfo `json:"consumers"`
-	Missing   []string        `json:"missing,omitempty"`
+	Consumers []*ConsumerInfo   `json:"consumers"`
+	Missing   []string          `json:"missing,omitempty"`
```

Comment:
Are Missing and Offline kinda the same?

Reply:
They are different. `Missing` is represented as "inaccessible" in the CLI, and means a R1 stream where the server hosting it is offline, or R3 where all 3 servers are offline, or if there's currently no leader.

`Offline` means a 2.11 server came up and recognized it can't support 2.12 features for a stream/consumer. That reports the stream/consumer name as the key, and the value contains the error message containing the API level that that asset requires.

We do populate the `Missing` with all entries in `Offline` as well, since on the one hand they _are_ technically missing in that case, but mostly because older tooling will not recognize `Offline` until upgraded.

Reply:
Unsupported is one possible reason for being offline. I imagine we can handle other reasons for being offline this way also - like subject clashes, corruption egc